src/components/UncategorizedNotification.vue

```diff
@@ -0,0 +1,32 @@
+<template lang="pug">
+div
+  // TODO: Add some way to disable this notification, probably by making the ratio threshold configurable
```

Comment:
Raised the ratio threshold to 20%, might be high enough to not need a disable feature (for now).